README.md

```diff
@@ -311,6 +311,31 @@ To verify:
 $ cosign verify-attestation --key cosign.pub <image>
 ```
 
+##### Validate In-Toto Attestations
+
+`cosign` has support of validating In-toto Attestations against `CUE` and `Rego` policies.
+
+Let's start with giving a brief introduction to `CUE` and `Rego` who might not be familiar with these.
+
+* [CUE](https://cuelang.org) is an open-source data validation language and inference engine with its roots in logic programming.
+
+* [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) was inspired by Datalog, which is a well understood, decades old query language. Rego extends Datalog to support structured document models such as JSON.
```

Comment:
🤔 I'd cut this down a bit:
```suggestion
* [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) was inspired by Datalog, and supports structured document models such as JSON.
```

Really wouldn't want to give the impression that CUE's logic foundation was _not well understood_ or anything. Less is more 😄